dep.cc

```diff
@@ -534,7 +536,7 @@ class DepBuilder {
 
   shared_ptr<Rule> first_rule_;
   unordered_map<Symbol, DepNode*> done_;
-  unordered_set<Symbol> phony_;
+  set<Symbol> phony_;
```

Comment:
I think this won't change build.ninja? This class does not iterate over |phony_|.